packages/netlify-cms-core/src/backend.ts

```diff
@@ -782,7 +780,6 @@ export class Backend {
     const updatedOptions = { unpublished, status };
     const opts = {
       newEntry,
-      parsedData,
```

Comment:
This is good cleanup. I think we can remove https://github.com/netlify/netlify-cms/blob/174d86f0a083a8c89e832962e3bdfe4163def428/packages/netlify-cms-backend-test/src/implementation.ts#L229 and https://github.com/netlify/netlify-cms/blob/8f1b325809a63bdc8915ae534c5686643911c038/packages/netlify-cms-lib-util/src/implementation.ts#L55 too right?

Reply:
Yup, we need to remove those as well.